.github/workflows/release.yml

```diff
@@ -5,7 +5,6 @@ on:
   push:
     branches:
       - main
-      - ENG-6843/shopify-cart-1.0.0
```

Comment:
I don't think we can remove this until #278.

Reply:
My bad, should be fixed now! Thank you @NWRichmond